### ISSUE TITLE
Repository refactor, folder structure & more

### ISSUE THREAD

**Issue by vagelisp** [wceu/wceu2023#7]
Hello all, 
I made a small refactor on the repository to have a better DevX and make life easier ( hopefully ) after the small chat I had with @Ioanna-Mitsani

 - Added [@wordpress/scripts](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/) to compile and Lint our SASS files while respecting [CSS Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/)
 - Switched to [SCSS syntax](https://sass-lang.com/documentation/syntax) 
 - Restructured the SASS files/folders
 - Added more info on the readme file
 - A few more edits/tweaks here and there - it's late can't track everything 😬 -

Some may find this too much, and I am not expecting this to be merged immediately.
Hopefully, the team will review it and then we can chat about it.
Again please don't merge before we have a chat with the team.

The PR can be found here #6 
Thanks

**Comment by vagelisp** [comment]
I will merge the #6 PR to go with this, as we all agreed yesterday that it's OK.
I added the old output.css file as a failsafe; we can remove that in the following days.

**Comment by vagelisp** [comment]
The old file's path has been changed, so I adjusted the path on the website. Everything seems to be working fine. 
I'll do some more testing and import the new file.

I am just documenting the steps/actions here in case anyone wants to follow/comment/change something.

Thanks.

**Comment by vagelisp** [comment]
I removed some of the CSS we had on "Additional CSS." 
Some of the rules are causing conflicts (i.e. with the menu as there's a global ul style rule ), and some were duplicates since Ioana added typography on git. I also extended the heading rules a bit to make them global ( ofc we can override when needed ). 

Also, I switched from the output.css to the new CSS file so I am moving this into the backup folder.
